### DISCUSSION TITLE
There are duplicate feed messages on the homepage

### DISCUSSION THREAD

**Discussion by rentiansheng** [community/community#77143] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/3955934/ba25e91a-b6a3-4b07-bcd0-a8a613023c97)
Suspected:
1. click star first.
2. cancel star

**Comment by dronelektron** [comment]
I have the same problem:

![изображение](https://github.com/community/community/assets/3470142/dcac1311-1221-4af5-a9ca-645206f06060)

**Reply by ufrshubham** [reply]
I am also seeing the same problem
![image](https://github.com/community/community/assets/33748002/62fc3d55-8daf-4b60-9229-3dc2164b4e06)

**Comment by Prayforhanluo** [comment]
same problem

**Comment by SNWCreations** [comment]
same
![image](https://github.com/community/community/assets/59989459/d1b643e0-3712-412a-9f8c-ea031b6d1573)

**Comment by SleeplessByte** [comment]
![image](https://github.com/community/community/assets/1964376/35df0bcd-ea5b-4d1e-af7f-f16fbc30e59e)

4x...
